source/clear-linux/tutorials/docker/docker.rst

```diff
@@ -79,7 +79,7 @@ packages to run Docker images as containers.
 Congratulations! At this point, you have a working installation of Docker
 on |CL|. You are ready to start using container images on your system.
 
-Integration with Kata Containers (optional)
+Integration with Kata Containers\* (optional)
 ********************************************
```

Comment:
Q: Oh lovely Sphinx, how many asterisks do you require? 
A: Just one more, please. 
Please add one more on Ln 82 to resolve build error.